<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>kktools</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="leftBarWin" native="true">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="10,3">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="NavTreeView" name="treeView">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="NoticeView" name="noticeView" native="true">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <widget class="QLabel" name="noticeText">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>200</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
         <widget class="QLabel" name="noticePic">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>200</width>
            <height>180</height>
           </rect>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="componentWin" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>打开</string>
    </property>
    <addaction name="actionsfsd"/>
    <addaction name="separator"/>
    <addaction name="actionb"/>
    <addaction name="separator"/>
    <addaction name="actionc"/>
    <addaction name="separator"/>
    <addaction name="actiond"/>
    <addaction name="separator"/>
    <addaction name="actionquanjupeizhi"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionsfsd">
   <property name="text">
    <string>a</string>
   </property>
  </action>
  <action name="actionb">
   <property name="text">
    <string>b</string>
   </property>
  </action>
  <action name="actionc">
   <property name="text">
    <string>c</string>
   </property>
  </action>
  <action name="actiond">
   <property name="text">
    <string>d</string>
   </property>
  </action>
  <action name="actionquanjupeizhi">
   <property name="text">
    <string>全局配置</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="iconText">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Help</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavTreeView</class>
   <extends>QListView</extends>
   <header>nav_tree_view.h</header>
  </customwidget>
  <customwidget>
   <class>NoticeView</class>
   <extends>QWidget</extends>
   <header>notice_view.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
